docs/api/misc.md

```diff
@@ -76,3 +76,35 @@ type Location = 'body' | 'cookies' | 'headers' | 'params' | 'query';
 
 Represents one of the request locations: `req.body`, `req.cookies`, `req.headers`, `req.params` and
 `req.query`.
+
+### `matchedData`
```

Comment:
Can you move these next to `matchedData`'s own documentation, in https://github.com/express-validator/express-validator/blob/master/docs/api/matched-data.md? Maybe under a "TypeScript usage" header.

Thanks!

Reply:
Sure thing! Thanks!